src/main/java/io/confluent/connect/jdbc/TimestampIncrementingTableQuerier.java

```diff
@@ -60,15 +60,17 @@
   private Long timestampOffset;
   private String incrementingColumn;
   private Long incrementingOffset = null;
+  private Long timestampDelay;
 
   public TimestampIncrementingTableQuerier(QueryMode mode, String name, String topicPrefix,
                                            String timestampColumn, Long timestampOffset,
-                                           String incrementingColumn, Long incrementingOffset) {
+                                           String incrementingColumn, Long incrementingOffset, Long timestampDelay) {
     super(mode, name, topicPrefix);
     this.timestampColumn = timestampColumn;
     this.timestampOffset = timestampOffset;
     this.incrementingColumn = incrementingColumn;
     this.incrementingOffset = incrementingOffset;
+    this.timestampDelay = timestampDelay == null ? 0L : timestampDelay;
```

Comment:
Not a big deal, but if we're going to resolve this to 0 anyway, we could just force this with the option itself (which you already do) and pass in a `long` instead of `Long`. For the other fields, the fact that they are `null` is used to determine the mode we're in, whereas here it is not.